google_cloudsql_mysql/versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.0"
```

Comment:
@cvalaas updated the version constraint today, change makes sense for >= 4.48.0 only